packages/substrate/src/fungible.ts

```diff
@@ -0,0 +1,158 @@
+import type {
+  Domainlike,
+  Environment,
+  SubstrateConfig,
+  SubstrateResource,
+} from '@buildwithsygma/core';
+import { LiquidityError, Config, FeeHandlerType, ResourceType } from '@buildwithsygma/core';
+import type { ApiPromise, SubmittableResult } from '@polkadot/api';
+import type { SubmittableExtrinsic } from '@polkadot/api-base/types';
+import { BN } from '@polkadot/util';
+
+import { BaseTransfer } from './base-transfer.js';
+import type { Fungible, SubstrateFee, Transfer } from './types.js';
+import {
+  deposit,
+  getBasicFee,
+  getFeeHandler,
+  getPercentageFee,
+  getLiquidity,
+} from './utils/index.js';
+
+export type SubstrateAssetTransferRequest = {
+  sourceDomain: Domainlike;
+  destinationDomain: Domainlike;
+  sourceNetworkProvider: ApiPromise;
+  resource: string | SubstrateResource;
+  amount: bigint;
+  destinationAddress: string;
+};
+
+async function checkDestinationFungibleHandler(
```

Comment:
I think this is an equivalent of EVM `getLiquidity`, we can remove this for now as we have a separate task for this

Reply:
FYI we are tracking this issue here https://github.com/sygmaprotocol/sygma-sdk/issues/383